tests/jvm/src/test/scala-2.12/scala/meta/tests/semanticdb/ExpectSuite.scala

```diff
@@ -17,6 +17,7 @@ import scala.meta.io.Classpath
 import scala.meta.io.AbsolutePath
 import scala.meta.tests.cli._
 import munit.FunSuite
+import munit.internal.difflib.DiffUtils
```

Comment:
`Diffutils` from munit work and I don't think we plan to change it at any point in the future. What do you think @olafurpg ?